packages/tldraw/src/lib/utils/shapes/shapes.ts

```diff
@@ -18,3 +18,10 @@ export function getTextLabels(geometry: Geometry2d) {
 
 	return []
 }
+
+export function maybeSnapToGrid(point: Vec, editor: Editor): Vec {
```

Comment:
very handy, i'd say this should be a `@public` fn

Reply:
ah nvm, i just saw this is also in `packages/editor/src/lib/editor/tools/BaseBoxShapeTool/children/Pointing.ts` - let's just have one version of this

Reply:
Thanks, couldn't figure out a way to use it across packages!